as-ma/as_ma/main.py

```diff
@@ -94,16 +95,17 @@ def write(self, reason, value):
 
         return
 
+    def _update_psconn_status(self):
+        for family, maepics in App.ma_devices.items():
+            reason = maepics.maname + ':PSConnStatus-Mon'
+            value = 1 if maepics.connected else 0
```

Comment:
isn't it better to use the enum property defined in `siriuspy.csdevice` instead of number literals?

Reply:
It is. I used literals for fast-implementation testing and latter forgot to replace it with `csdevice` constants. thanks.

Reply:
done.